bin/encodings

```diff
@@ -52,6 +52,7 @@ when "big5"         then lists("big5", 0...0x10000, Encoding::Big5)
 when "cp850"        then table(Encoding::CP850)
 when "cp852"        then table(Encoding::CP852)
 when "cp855"        then table(Encoding::CP855)
+when "cp874"        then table(Encoding::CP874)
```

Comment:
Typically we're only putting the main constants in here, not the aliases. `Encoding::Windows_874` is the main one, `Encoding::CP874` is the alias.